third_party/ti_simplelink_sdk/ti_simplelink_sdk.gni

```diff
@@ -63,7 +66,7 @@ template("ti_sysconfig") {
     }
   }
 
-  action("${target_name}_gen") {
+  pw_python_action("${target_name}_gen") {
```

Comment:
what's the advantage of using a pigweed action instead of a gn action here?

Reply:
It squelches the output, unless the script fails.